samples/hybrid-connections/java/simple-websocket-demo/README.md

```diff
@@ -0,0 +1,90 @@
+# 'Simple' Web Socket Hybrid Connection Sample 
+
+First, as described in the description [Here](https://github.com/bainian12345/azure-relay/tree/java-samples/samples/hybrid-connections/java), you will need to retrieve different parameters from the enviroment variable `RELAY_CONNECTION_STRING` for connection and authentication purposes:
+
+```	java
+	static final String CONNECTION_STRING_ENV_VARIABLE_NAME = "RELAY_CONNECTION_STRING";
+	static final Map<String, String> connectionParams = HybridConnectionUtil.parseConnectionString(System.getenv(CONNECTION_STRING_ENV_VARIABLE_NAME));
```

Comment:
>connectionParams = HybridConnectionUtil.parseConnectionString [](start = 34, length = 61)

Wait, why isn't RelayConnectionStringBuilder class used? It's a first class object for helping users parse this.

Reply:
Thought this class would be internal use only. It will work great here.

---
In reply to: [243472356](https://github.com/Azure/azure-relay/pull/37#discussion_r243472356) [](ancestors = 243472356)